modyn/supervisor/internal/pipeline_executor/evaluation_executor.py

```diff
@@ -103,12 +103,14 @@ def init_from_path(cls, pipeline_logdir: Path) -> "EvaluationExecutor":
         eval_state_config = EvalStateConfig.model_validate_json((snapshot_dir / "eval_state.yaml").read_text())
         context = pickle.loads((snapshot_dir / "context.pcl").read_bytes())
 
+        grpc_handler = GRPCHandler(eval_state_config.config.model_dump(by_alias=True))
+        grpc_handler.init_cluster_connection()
```

Comment:
Haha, I also just fixed that locally :D

Reply:
had it in my other branch for some time now, but this seems to be a good place to sneak it in :)